package.json

```
@@ -17,6 +18,8 @@
   "dependencies": {
     "@docusaurus/core": "3.8.1",
     "@docusaurus/preset-classic": "3.8.1",
+    "@fontsource/poppins": "^5.2.6",
+    "@fontsource/inter": "^5.2.6",
```

```suggestion
    "@fontsource/poppins": "5.2.6",
    "@fontsource/inter": "5.2.6",
```

Otherwise renovate will just create a PR afterwards to pin these.